### PR TITLE
Chapter 1/Homework 1

### PR DESCRIPTION
**Pull request by aysenti**
Here's my attempt at homework 1 (the bmi one), and homework 2 which I'm assuming was just forking, branching, and then doing a PR.

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,27 @@ class Program
...
+
+            int cWeight = 68;
+            float cHeight = 1.55F;
+            float cBmi = cWeight / cHeight;
+            int cAge = 20;
+
+            int bWeight = 61;
+            float bHeight = 1.41F;
+            float bBmi = bWeight / bHeight;
+            int bAge = 32;
```
Comment (jdphenix) [inline Src/BootCamp.Chapter/Program.cs]: Looking good so far! You're still needing to read in values from the console using `Console.ReadLine()`. 

    // getting a string
    Console.WriteLine("Enter a string.");
    string value = Console.ReadLine();

There's still a bit to do for numeric variables. Take a look at the [Finding missing pieces (Console)](https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/1And2#finding-missing-pieces-console) section for ideas.

**Review by jdphenix — CHANGES_REQUESTED** [review]
Looking good so far. Don't forget the spec asks you to implement prompting for a few key data values and outputting them back out, then repeat it again for another person. 

Ref (https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/1And2#code)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -23,10 +23,35 @@ static void Main(string[] args)
...
+            Console.WriteLine("Please enter your weight now. ");
+            int yourWeight = Convert.ToInt32(Console.ReadLine());
+            double yourBMI = yourWeight / yourHeight;
+            Console.WriteLine("Alright " + yourName + ", " + "you are " + yourAge + " years old, and your BMI based off your provided measurements is " + yourBMI);
+
+            Console.WriteLine("Press any key to continue...");
+
             Console.ReadLine();
+            Console.Clear();
+            goto Start;
```
Comment (jdphenix) [inline Src/BootCamp.Chapter/Program.cs]: Yipe! 

Consider a simple way of asking for two people's data without using `goto`. One way might be: 

    for (int i = 0; i < 2; i++)
    {
      // Ask and output here
    }

If you haven't seen a [for loop](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/for) before, that's fine! Basically, what this does is declare a variable `i`, makes sure that `i < 2`, then runs the code inside the brackets, then adds 1 to `i` and repeats. 

I definitely want to discourage to use of `goto`, in any circumstance.

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -23,10 +23,35 @@ static void Main(string[] args)
...
+            Console.WriteLine("Hello " + yourName);
+
+            Console.WriteLine("Please enter your age now. ");
+            int yourAge = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Please enter your height now. ");
+            double yourHeight = double.Parse(Console.ReadLine());
+            Console.WriteLine("Please enter your weight now. ");
+            int yourWeight = Convert.ToInt32(Console.ReadLine());
+            double yourBMI = yourWeight / yourHeight;
+            Console.WriteLine("Alright " + yourName + ", " + "you are " + yourAge + " years old, and your BMI based off your provided measurements is " + yourBMI);
```
Comment (jdphenix) [inline Src/BootCamp.Chapter/Program.cs]: Note the spec for this assignment would have you ask for name and surname.

**Review by jdphenix — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/Program.cs (hunk 3 above)
Comment (aysenti) [reply Src/BootCamp.Chapter/Program.cs]: Alright I reworked it to ask for surname and to also not use goto

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -1,57 +1,57 @@
...
+                string surname = Console.ReadLine();
+                Console.WriteLine("Hello " + firstName + " " + surname);
+
+                Console.WriteLine("Please enter your age now. ");
+                int yourAge = Convert.ToInt32(Console.ReadLine());
+                Console.WriteLine("Please enter your height now. ");
+                double yourHeight = double.Parse(Console.ReadLine());
+                Console.WriteLine("Please enter your weight now. ");
+                int yourWeight = Convert.ToInt32(Console.ReadLine());
+                double yourBMI = yourWeight / yourHeight;
```
Comment (jdphenix) [inline Src/BootCamp.Chapter/Program.cs:44]: I should have caught this on the first review. bmi is weight over height squared.

**Review by jdphenix — APPROVED** [review]
Overall happy with this program. Keep in mind sticking to what's asked of the spec and coding style (extra empty lines).

File: Src/BootCamp.Chapter/Program.cs (hunk 4 above)
Comment (aysenti) [reply Src/BootCamp.Chapter/Program.cs:44]: Oh my bad I also should have noticed that when I looked up the formula

**Pull request closed without merging**